docs/content/build-and-deploy.md

```diff
@@ -60,10 +60,13 @@ Use the [Inspector](https://docs.improbable.io/reference/latest/shared/operate/i
 
 ### Cloud deployment
 To start a cloud deployment, follow these steps: 
-1. Upload your server-worker and client-worker assemblies. To do this, in a terminal window, from any directory, run `spatial cloud upload <assembly-name>` from the [`spatial` CLI](https://docs.improbable.io/reference/latest/shared/glossary#the-spatial-command-line-tool-cli). 
+1. Specify the SpatialOS project name you want to deploy to. To do this, locate the spatialos.json file in the root folder of your SpatialOS project and edit the file so the name field matches your SpatialOS project name. You can find your project name in the [SpatialOS Console](https://console.improbable.io/). It’ll be something `like beta_someword_anotherword_000`. (To set up your project in SpatialOS, follow the instructions in the [Setup guide](../setup-and-installing.md).)
+
+ Ensure you set the `name` field in the `spatialos.json` to a SpatialOS project that you have access to. You can find your project name in the [SpatialOS Console](https://console.improbable.io/). It’ll be something like `beta_someword_anotherword_000`.
+2. Upload your server-worker and client-worker assemblies. To do this, in a terminal window, from any directory, run `spatial cloud upload <assembly-name>` from the [`spatial` CLI](https://docs.improbable.io/reference/latest/shared/glossary#the-spatial-command-line-tool-cli). 
```

Comment:
"from any directory" I would change this to "from any directory in your project". I know that this should be obvious, but just in case.